### DISCUSSION TITLE
Problems with the bottom and top bar. ¿Can you help me?

### DISCUSSION THREAD

**Discussion by perojas3** [gsantner/markor#2461] [Q&A] [answered]
I have a problem. Don't know if is my fault, or is a bug. Hope someone can help me.
The first issue is between the top bar of the phone (the one with the date, the notifications, signal data, and the remaining battery of the phone), and the top bar of the text editor (contains the title of the file, undo, redo, view markdown, save, look up and the 3 dots for more).
When I use markor to edit a document, the editor top bar is drawn below the phone bar, so I cannot use the editor bar:
![Screenshot_20241025-201904](https://github.com/user-attachments/assets/2e96e815-8556-4235-be53-fa5799089131)

This problem does not happen when editing a to-do:
![Screenshot_20241025-202444](https://github.com/user-attachments/assets/b4ff7836-cad8-4b09-bbb8-ebde795490b6)

The second issue is in the same view with the bottom bar. That when I open the keyboard, the bottom bar does not raise and remains behind the keyboard, so I have to withdraw the keyboard to see the bottom bar and the bottom of the document. Like this:
![Screenshot_20241025-202031](https://github.com/user-attachments/assets/b0462f55-6dca-4574-8090-89e311f0963f)

As with the previous issue, it does not happen when editing a to-do. Is like, when editing an external document, the editor UI ignores the phone boundaries, but not while editing a to-do.
I'm using a pixel 7 phone with Graphene OS, with markor v2.12.5.

Hope you can help me.
Thanks in advance.

**Comment by gsantner** [comment, marked as answer]
Check closed issues.

**Reply by perojas3** [reply]
Thanks a lot for the fast answer.
I cannot believe I missed all those close threads.
Also, noticed you are answering this to everyone who's asking this, like me, and no one is thanking you.
So thanks for that too.
